### DISCUSSION TITLE
Toggle feature for companion app actions.

### DISCUSSION THREAD

**Discussion by dorji-tshering** [dailydotdev/daily#590] [Feature Requests]
If you visit [https://medium.com/geekculture/the-dark-truth-about-vpn-cb2d8d9735ad](url), and if you open the comment section at the end of the article, you can see that the companion app overlaps the comment contents. So if there is a toggle feature to show and hide the companion app(show the app initially), I think it will help improve user experience. Of course majority of the sites don't seem to use full width for the contents but for some like the one mentioned, a toggle feature can really do wonders to the experience of the readers. For instance, I had to scroll the comment section that I want to read either to top or below the companion app. Also I would like to report (may not necessarily a bug) a user experience issue . On the same page, when the comment section is opened on the right side, the companion app doesn't seem to work(for instance the options are not showing upon clicking the option icon). However when the comment section is closed, it works normal. 

Finally, thanks to every one involved in developing the daily.dev!

**Comment by idoshamun** [comment]
Hi!

You're right that a valid edge case that we should support. I'll log it down for our next planning meeting :)
